.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "latest"
+  ],
+  "plugins": [
+    "add-module-exports",
+    "transform-flow-strip-types"
```

Comment:
Since we don't have flow types, we also don't need to strip them! Other than that, this looks great!